collection-scripts/gather_haproxy_config

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="must-gather"
+HAPROXY_CONFIG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/haproxy"}
+
+mkdir -p "${HAPROXY_CONFIG_PATH}"/
+
+function gather_haproxy_config {
+    echo "INFO: Gathering HAProxy config files"
+    INGRESS_CONTROLLERS=$(oc get ingresscontroller -n openshift-ingress-operator --no-headers -o custom-columns=":metadata.name")
+    for IC in ${INGRESS_CONTROLLERS}; do
+        POD_STRING=$(oc get pods -n openshift-ingress --no-headers -o custom-columns=":metadata.name" --selector ingresscontroller.operator.openshift.io/deployment-ingresscontroller="${IC}")
+        PODS=($POD_STRING)
```

Comment:
Just a suggestion:  You could use `-o name`, although that has the disadvantage that you need to strip off the leading resource qualifier (e.g., `oc get ingresscontrollers -o name` prints "ingresscontroller.operator.openshift.io/default", and you want just "default"):
```suggestion
    INGRESS_CONTROLLERS=$(oc get ingresscontroller -n openshift-ingress-operator -o name)
    for IC in ${INGRESS_CONTROLLERS}; do
        PODS=($(oc get pods -n openshift-ingress -o name -l ingresscontroller.operator.openshift.io/deployment-ingresscontroller="${IC#*/}"))
```

Reply:
Other scripts in `must-gather` seem to more often use the `columns=...` approach, so i'll stick with the current approach for consistency sake. That being said, I will wrap the line in `()` to assign the values as an array. Thanks!